pixi.toml

```diff
@@ -0,0 +1,67 @@
+# Pixi is a package management tool for developers.
+# Before running a task, pixi ensures that all listed dependencies are installed first.
+#
+# Run the tasks with `pixi run TASK`, e.g. `pixi run codegen`.
+# Extra CLI argument after `pixi run TASK` are passed to the task cmd.
+# Lists tasks with `pixi task list`
+#
+# Type `pixi shell` to enter the pixi environment.
+#
+# https://prefix.dev/docs/pixi/overview
+
+[project]
+name = "rerun"
+authors = ["Jeremy Leibs <jeremy@rerun.io>"]
```

Comment:
I mean I'm okay with Jeremy owning this, but it's more appropriate to put opensource@rerun.io :)